pinot-common/src/main/java/org/apache/pinot/common/response/broker/BrokerResponseStats.java

```diff
@@ -33,7 +33,7 @@
 //  same metadataKey
 // TODO: Replace member fields with a simple map of <MetadataKey, Object>
 // TODO: Add a subStat field, stage level subStats will contain each operator stats
-@JsonPropertyOrder({"exceptions", "numBlocks", "numRows", "stageExecutionTimeMs", "stageExecutionUnit",
+@JsonPropertyOrder({"requestId", "exceptions", "numBlocks", "numRows", "stageExecutionTimeMs", "stageExecutionUnit",
```

Comment:
requestID generated in v1 engine doesn't include a hashed encoding of the broker address. so essentially this could cause confusion (e.g. requestID starts with 0 and you might get another 0 when hitting a different broker)

see: #10789 for more info

Reply:
Yeah that's a problem, I'll submit another PR to reuse `MultiStageRequestIdGenerator` in v1 engine.